templates/4a_compensate_image_data.ipynb

```diff
@@ -14,7 +14,7 @@
    "id": "237267d0",
```

Comment:
<p>I have found that there are times in which I want to test my matrix on ~10 FOVs, but my runs for a single slide were split up so for example I have 3 runs each with 7 FOVs. This meant that I had to run through the extraction and normilization process twice, once to get 7 FOVs from one run and a second time to get the last 3 FOVs from another run. I then combined the FOVs into one folder after normilization, but all of that to say, I think there should be a note on how to get 10 FOVs if your FOVs are split across multiple runs. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I think one run of 3 and another of 7 is pretty rare. The function currently takes the supplied number from each run, so assuming the user has at least 3 runs, taking 3 from each would get the job done. </p>